src/main/java/org/opensearch/knn/index/vectorvalues/KNNBinaryVectorValues.java

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
Could you update this licensing?
```
/*
 * Copyright OpenSearch Contributors
 * SPDX-License-Identifier: Apache-2.0
 */
```

Reply:
Seems like the change of license just went, after these files are created. Let me fix it now.